app/cast/[username]/[hash]/page.tsx

```diff
@@ -21,9 +24,53 @@ export const metadata: Metadata = {
   },
 };
```

Comment:
if you remove the `export` this variable becomes unused right?

Reply:
no i am returning default metadata if cast is not found

 catch (error) {
    console.error('Failed to generate metadata:', error);
    return metadata;
  }

Reply:
so that the og image is not empty.